### DISCUSSION TITLE
How to set up a demo from local

### DISCUSSION THREAD

**Discussion by robbynie** [athensresearch/athens#2197] [General]
I'd like to setup a demo like: https://web.athensresearch.org/ from local server.
Any advices? Thanks.

**Comment by lisachev** [comment]
Beta37 includes a web client by default. If you follow the installation instructions, you will end up with a web client served by the same nginx server
https://athensresearch.github.io/docs/developer_guide/self-hosted#server
at least the Docker path will lead you to it.

**Reply by ithakaa** [reply]
I tried this many times with any success at all
